sdk/rust/src/consensus/engine.rs

```diff
@@ -32,7 +32,7 @@ pub enum Update {
     BlockCommit(BlockId),
 }
 
-#[derive(Default, Debug, Eq, Hash, PartialEq)]
+#[derive(Clone, Default, Debug, Eq, Hash, PartialEq, PartialOrd)]
```

Comment:
These derives should go in a separate commit.

Reply:
separated